scripts/src/public/services/migration.ts

```diff
@@ -27,18 +29,12 @@ export const accountStatus = async function(req: express.Request, res: express.R
 
 	const app_blockchain_version = await getAppBlockchainVersion(appId);
 
-	const isBurnedRes = await client.get(`${ getConfig().migration_service }/accounts/${ publicAddress }/status`, {
-		validateStatus: status => {
-			return status < 500; // Reject only if the status code is greater than or equal to 500
-		}
-	});
-	if (isBurnedRes.status === 404) {
-		isBurnedRes.data.is_burned = true; // Wallet doesn't exist
-	} else if (isBurnedRes.status > 400) {
-		logger().info(`Something went wrong, migration service response`, isBurnedRes);
-		throw Error(`Something went wrong, migration service response: ${ JSON.stringify(isBurnedRes.data) }`);
+	const wallet = await WalletApplication.findOne({ walletAddress: publicAddress });
+	if (!wallet) {
+		throw NoSuchWallet(publicAddress);
```

Comment:
@doodyparizada WDTY, should this throw or if the `app_blockchain_version` is 3 and no wallet maybe return `shouldMigrate: true`.

Reply:
pretty sure it should behave as before when receiving `404`:
```javascript
if (isBurnedRes.status === 404) {
		isBurnedRes.data.is_burned = true; // Wallet doesn't exist
```